samples/meetings-transcription/csharp/MeetingTranscription/Helpers/GraphHelper.cs

```diff
@@ -0,0 +1,92 @@
+﻿// <copyright file="GraphHelper.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace MeetingTranscription.Helpers
+{
+    using Microsoft.Extensions.Options;
+    using Microsoft.Identity.Client;
+    using MeetingTranscription.Models.Configuration;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http.Headers;
+    using System.Threading.Tasks;
+    using System.Net.Http;
+    using MeetingTranscription.Models;
+    using Newtonsoft.Json.Linq;
+
+    public class GraphHelper
+    {
+        /// <summary>
+        /// Stores the Azure configuration values.
+        /// </summary>
+        private readonly IOptions<AzureSettings> azureSettings;
+
+        public GraphHelper(IOptions<AzureSettings> azureSettings)
+        {
+            this.azureSettings = azureSettings;
+        }
+
+        /// <summary>
+        /// Gets application token.
+        /// </summary>
+        /// <returns>Application token.</returns>
+        public async Task<string> GetToken()
+        {
+            IConfidentialClientApplication app = ConfidentialClientApplicationBuilder.Create(this.azureSettings.Value.MicrosoftAppId)
+                .WithClientSecret(this.azureSettings.Value.MicrosoftAppPassword)
+                .WithAuthority($"https://login.microsoftonline.com/{this.azureSettings.Value.MicrosoftAppTenantId}")
+                .WithRedirectUri("https://daemon")
+                .Build();
+
+            // TeamsAppInstallation.ReadWriteForChat.All Chat.Create User.Read.All TeamsAppInstallation.ReadWriteForChat.All
+            string[] scopes = new string[] { "https://graph.microsoft.com/.default" };
+            var result = await app.AcquireTokenForClient(scopes).ExecuteAsync();
+
+            return result.AccessToken;
+        }
+
+        /// <summary>
+        /// Gets the meeting transcript for the passed meeting Id.
```

Comment:
nit: Gets **_all_** the meeting transcripts for the passed meeting Id.

Reply:
Done